dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/JarLinkageReport.java

```diff
@@ -141,29 +138,29 @@ public String toString() {
    * @return the total number of classes that refer to missing or inaccessible members
    */
   public int getErrorCount() {
-    return getCauseToSourceClasses().size();
+    return getTargetToSources().size();
   }
   
   /**
-   * @return the total number of missing or inaccessible members
+   * @return the total number of missing or inaccessible targets
```

Comment:
This isn't true, is it? You can have more than one LEC for a target class. This is the total number of errors (reasons for problematic symbols) independent of how many source classes trigger that error.

The method name is wrong too.